3741/3/models/review.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class Review < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :place
```

Comment:
Add empty line between associations and validations

Reply:
added